pkg/terraform/stages/aws/stages.go

```diff
@@ -41,3 +52,37 @@ func customBootstrapDestroy(s stages.SplitStage, directory string, terraformDir
 		"failed to destroy bootstrap",
 	)
 }
+
+func extractAWSLBConfig(s stages.SplitStage, directory string, config *types.InstallConfig) ([]string, []string, error) {
```

Comment:
If the user is going to supply load balancer (internal and external) information, why do we need to extract it from terraform? Shouldn't terraform skip any creation of this data, so we would already know this information and can populate any structure without the output of terraform right ?

Reply:
We donot expect the user to provide LB information anymore. They will not pre-create these resources. Instead, Installer will create these resources like a regular IPI install and only skip route53 configuration.